docs/Technical-Documentation/Architecture-Decision-Record/018-versioning-and-releases.md

```diff
@@ -0,0 +1,59 @@
+# 18. Versioning and Release strategy
+
+Date: 2022-04-05 (yyyy-mm-dd)
+
+## Status
+
+Accepted
+
+## Context
+As discussed briefly in [deployment-flow](./008-deployment-flow.md), we will need a strategy for our versioning and releases.
+
+## Decision
+
+### Gitflow
+We will use the industry standards "gitflow" to handle release branching above our [git-workflow](./009-git-workflow.md). You can read more about gitflow [here](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) and [here](https://datasift.github.io/gitflow/IntroducingGitFlow.html). As shown below, the raft team will merge any relevant feature branches of the form `feat/###-branch-description` into the `develop` branch. From this branch, we will split off a release-specific branch of the form `release/v#.#.#`. This release branch can be merged to `HHS/TANF-app:main`  which will deploy to staging site for review and testing. Once this release is fully ready, `HHS/TANF-app:main` will be merged into `HHS/TANF-app:master` which should contain only finalized merge commits of the release versions and will deploy to production.
```

Comment:
@lfrohlich highlighting this section as the important change discussed today at tech sync. cc: @abottoms-coder @stevenino